src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -6715,10 +6715,6 @@ static int Main()
 ";
             var semanticInfo = GetSemanticInfoForTest<InitializerExpressionSyntax>(sourceCode);
             Assert.Null(semanticInfo.Type);
-            //Assert.Equal("int[]", semanticInfo.Type.ToString());
-            //Assert.Equal("int[]", semanticInfo.ConvertedType.ToString());
-            //Assert.Equal(ConversionKind.Identity, semanticInfo.ImplicitConversion.Kind);
-            //Assert.True(semanticInfo.IsCompileTimeConstant);
```

Comment:
There's little documentation on what `IsCompileTimeConstant` entails in the semantic model. I'm surprised this would be considered a constant. Does this differ if anything other than a literal is included in the initializer?

Reply:
This is a deletion of wrong code, so the question doesn't really apply to this change.

---
In reply to: [171347438](https://github.com/dotnet/roslyn/pull/23921#discussion_r171347438) [](ancestors = 171347438)

Reply:
Ugh, I can't believe I missed that 🙃